webpay/spa/views.py

```diff
@@ -40,4 +39,6 @@ def index(request, view_name=None, start_view=None):
                          str(emails))
                 set_user(request, emails[0], verified=False)
                 ctx['mkt_user'] = True
+
+    ctx['super_powers'] = request.session.get('super_powers', False)
```

Comment:
Any reason this was moved? Right after `ctx` is created seems like a good place to set this. Possibly even in the same statement.